data/crac-io/crac-io-json/pom.xml

```diff
@@ -75,6 +75,12 @@
             <artifactId>powsybl-commons-test</artifactId>
             <version>${powsybl.core.version}</version>
         </dependency>
+        <dependency>
+            <groupId>com.powsybl</groupId>
+            <artifactId>open-rao-crac-creator-api</artifactId>
+            <version>5.2.0-SNAPSHOT</version>
+            <scope>compile</scope>
+        </dependency>
```

Comment:
what you are trying to do here is to create two files that can read and write RaUsageLimits (CRAC creation parameters & CRAC). It just happens that both are in json format, and both are written the same way, in the current version.
In order to avoid code duplication, you created a depedency which should not exist.
I'd rather have the code duplication, or the common code moved to a third "utils" package

Reply:
I understand.
I would like to avoid duplicating the code and rather moving it into the package `crac-creation-util`. 
The only issue is that doing so creates a circual dependency for only 1 reason : `OpenRaoImportException` needs `ImportStatus` which is located here `data.craccreation.creator.api` :disappointed: .
Moving `ImportStatus`to the `crac-creation-util` would solve the issue and avoid code duplication.
But it would mean this PR modifies tons of files. 
Do you agree with this solution or should I just give up and duplicate the code?

Reply:
just duplicate the code since, from a functional point of view, there is no reason to always have the same